.github/workflows/continuous-integration.yml

```
@@ -1,4 +1,4 @@
-# GitHub Actions workflow specifically designed for running PHPUnit tests
+8.5# GitHub Actions workflow specifically designed for running PHPUnit tests
```

actionlint (1.7.8)

workflow is scalar node but mapping node is expected

(syntax-check)